src/js/modules/scrollToAnchor.js

```diff
@@ -0,0 +1,22 @@
+import { closeMenu } from './mobileMenu.js';
```

Comment:
move all the code to mobileMenu file it is still all about menu

Reply:
and rename file as just "menu"

Reply:
Done